f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -690,12 +694,16 @@ def _create(self, bigip, vs, listener, service):
         if listener['protocol'] == "HTTP" or \
            listener['protocol'] == "TERMINATED_HTTPS":
             self._create_http_profile(bigip, vs)
-        if 'bandwidth' in loadbalancer.keys() and \
-           loadbalancer['bandwidth']:
+
+        bandwidth = LoadBalancerManager.get_bandwidth_value(
```

Comment:
We can invoke this method via a LoadBalancerManager instance, right?

Reply:
as above